DEFRA/hapi-mustache-base-app
# Further gulpfile changes

1. Changed 'gulp build' task to copy assets and get scss working correctly.
2. Changed default 'gulp' task to execute the app using nodemon and then proxy using Browser Sync to fire up a Google Chrome browser and automatically refresh the browser when SCSS / HTML / JS assets are modified.
3. Other Gulpfile refactoring.

To test these changes, run the following tasks:
gulp build
gulp

Chrome should fire up at: http://localhost:3000/ and the page should refresh automatically whenever the application assets are modified.
